Non constexpr expressions in tests

Tests fail to compile due to non-`constexpr` parts:
```
[ 85%] Building CXX object test/CMakeFiles/test_portsofcall.dir/test_static_vector.cpp.o
oss/ports-of-call/test/test_span.cpp:567:24: error: constexpr variable 'pi' must be initialized by a constant expression
  567 |   constexpr const Real pi = std::acos(-1);
      |                        ^    ~~~~~~~~~~~~~
/opt/rh/gcc-toolset-12/root/usr/lib/gcc/x86_64-redhat-linux/12/../../../../include/c++/12/cmath:100:14: note: subexpression not valid in a constant expression
  100 |     { return __builtin_acos(__x); }
      |              ^
oss/ports-of-call/test/test_span.cpp:567:29: note: in call to 'acos(-1)'
  567 |   constexpr const Real pi = std::acos(-1);
oss/ports-of-call/test/test_span.cpp:569:18: error: constexpr variable 'd_gp' must be initialized by a constant expression
  569 |   constexpr Real d_gp = 2. * pi / static_cast<Real>(N);
      |                  ^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
oss/ports-of-call/test/test_span.cpp:569:30: note: initializer of 'pi' is not a constant expression
  569 |   constexpr Real d_gp = 2. * pi / static_cast<Real>(N);
      |                              ^
oss/ports-of-call/test/test_span.cpp:567:24: note: declared here
  567 |   constexpr const Real pi = std::acos(-1);
      |                        ^
2 errors generated when compiling for gfx942.
```

--- a/test/test_span.cpp
+++ b/test/test_span.cpp
@@ -1,7 +1,7 @@
-#include "ports-of-call/span.hpp"
 #include "ports-of-call/portability.hpp"
-#include <type_traits>
+#include "ports-of-call/span.hpp"
 #include <cmath>
+#include <type_traits>
 
 // ========================================================================================
 // © (or copyright) 2019-2024. Triad National Security, LLC. All rights
@@ -564,19 +564,17 @@ TEST_CASE("span portability", "[PortsOfCall::span]") {
   constexpr const std::size_t Nb = N * sizeof(Real);
   constexpr const Real tol = 1.0E-8;
 
-  constexpr const Real pi = std::acos(-1);
+  constexpr const Real pi = 3.141592653589793238462643383279502884;
 
   constexpr Real d_gp = 2. * pi / static_cast<Real>(N);
 
   std::vector<Real> h_gp(N);
-  for(auto i = 0; i < N; ++i)
-  {
+  for (auto i = 0; i < N; ++i) {
     h_gp[i] = -pi + static_cast<Real>(i) * d_gp;
   }
 
+  Real *p_a = (Real *)PORTABLE_MALLOC(Nb);
 
-  Real *p_a = (Real*) PORTABLE_MALLOC(Nb);
-  
   // device span, host span
   span d_s(p_a, N);
   span h_s(h_gp);
@@ -586,11 +584,13 @@ TEST_CASE("span portability", "[PortsOfCall::span]") {
 
   // integrate cos x dx over [-pi,pi]
   Real res_sum{0.0};
-  portableReduce( "integrate", 0, N, PORTABLE_LAMBDA(const int i, Real& rsum){ rsum += (std::cos(d_s[i])*d_gp) ;}, res_sum);
+  portableReduce(
+      "integrate", 0, N,
+      PORTABLE_LAMBDA(const int i, Real &rsum) { rsum += (std::cos(d_s[i]) * d_gp); },
+      res_sum);
 
   // require sum < tol
-  REQUIRE(std::abs(res_sum) < tol); 
-
+  REQUIRE(std::abs(res_sum) < tol);
 
   PORTABLE_FREE(p_a);
 }

--- a/test/test_span.cpp
+++ b/test/test_span.cpp
@@ -564,7 +564,7 @@ TEST_CASE("span portability", "[PortsOfCall::span]") {
   constexpr const std::size_t Nb = N * sizeof(Real);
   constexpr const Real tol = 1.0E-8;
 
-  constexpr const Real pi = 3.141592653589793238462643383279502884;
+  constexpr const Real pi = M_PI;
 
   constexpr Real d_gp = 2. * pi / static_cast<Real>(N);
 

